.tekton/timestamp-authority-push.yaml

```diff
@@ -92,11 +92,11 @@ spec:
       description: Skip optional checks, set false if you want to run optional checks
       name: skip-optional
       type: string
-    - default: "false"
+    - default: "true"
       description: Execute the build with network isolation
       name: hermetic
       type: string
-    - default: ""
+    - default: [{"path": ".", "type": "gomod"}, {"path": "./hack/tools", "type": "gomod"}]
```

Comment:
Same as above - add these values to `spec.params` instead of `spec.pipelineSpec.params`.